tests/ClientTest.php

```diff
@@ -334,10 +344,18 @@ public function testDsnFirstArgument()
     {
         $client = new Dummy_Raven_Client('http://public:secret@example.com/1');
 
-        $this->assertEquals(1, $client->project);
-        $this->assertEquals('http://example.com/api/1/store/', $client->server);
-        $this->assertEquals('public', $client->public_key);
-        $this->assertEquals('secret', $client->secret_key);
+        $project = new \ReflectionProperty($client, 'project');
```

Comment:
PHPUnit provides an `assertAttributeEquals` method to check whether an attribute (even private) equals a given value